docker/rockylinux9/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM mipsdkcontainer.azurecr.io/rockylinux9
```

Comment:
Is this registry publicly accessible? If not, we shouldn't be adding the setup here.

Reply:
It is going to be checked in MIP SDK repo.

For global OneDs I will raise separate PR.

Reply:
My concern is - if this registry is Microsoft internal, we are exposing a name of private/internal registry name "mipsdkcontainer.azurecr.io" in public repo (even though it is separate branch).  Why do this need to be added here?

Reply:
I see, But it seems other docker files  MIP SDK-specific OneDS branch also have a similar entries like for centos, ubuntu

FROM mipsdkcontainer.azurecr.io/centos8
FROM mipsdkcontainer.azurecr.io/ubuntu2004

FROM mipsdkcontainer.azurecr.io/library/centos:7

Reply:
All these flavors have similar entry

<img width="1063" alt="image" src="https://user-images.githubusercontent.com/122050221/233189038-f7315f4f-9222-4713-9729-9fe556370d6d.png">

Reply:
ok was just a suggestion. I don't have any issue as long as it is safe to expose these registries, and you are the better judge of that :)

Reply:
Just checked an Authentication is required if some accessing outside. Also, this aims for a user branch...not the main OneDS branch....so I think its less risky, that may be the reason Vitally made these changes